content/20.pharmaceuticals.md

```diff
@@ -372,6 +372,16 @@ Both doses were well tolerated with no serious events related to the antibody co
 Based on this data, the FDA granted an EUA for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk of developing severe disease [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-monoclonal-antibodies-treatment-covid-19].
 Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients [@clinicaltrials:NCT04426695].
 
+Sotrovimab is the most recent mAb to receive an EUA.
+It was identified in the memory B cells of a 2003 survivor of SARS [@doi:10.1038/nm1080] and was found to be cross-reactive with SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y].
+This cross-reactivity is likely attributable to conservation within the epitope, with 17 out of 22 residues conserved between the two viruses, four conservatively substituted, and one semi-conservatively substituted [@doi:10.1038/s41586-020-2349-y].
+In fact, these residues are highly conserved among sarbecoviruses [@doi:10.1038/s41586-020-2349-y].
+This versatility has led to it being characterized as a "super-antibody" [@doi:10.1038/s41587-021-00980-x], a potent, broadly neutralizing antibody [@doi:10.1038/nri.2017.148].
+Interim analysis of data from a clinical trial [@clinicaltrials:NCT04545060] reported high safety and efficacy of this mAb in 583 COVID-19 patients [@doi:10.1101/2021.05.27.21257096].
+Sotrovimab was found to be 85% effective in reducing progression to the primary endpoint, which was the proportion of patients hospitalized for more than 24 hours or dying within 29 days of receiving sotrovimab or a placebo.
```

Comment:
I made this suggested change directly in @rando2's add-mab branch. Not sure I was supposed to do that - sorry!

Changing this sentence to:
"Compared to placebo, sotrovimab was found to be 85% more effective in reducing progression to the primary endpoint, which was the proportion of patients who, within 29 days, were either hospitalized for more than 24 hours or died."

Reply:
Looks good to me!